aspnetcore/fundamentals/configuration/index/samples/3.x/ConfigSample/Options/PositionOptions.cs

```diff
@@ -3,6 +3,8 @@
     #region snippet
     public class PositionOptions
     {
+        public const string Position = "Position";
```

Comment:
I think this being a constant confuses things, even though I get the idea of not hardcoding a string. If I have an instance of `PositionOptions`, I might type `options.Position` with the help of intellisesne. One of the last things I'd expect that to be is an identifier used by the framework's configuration system.

Reply:
Let's review this later. Perhaps not use it when first used, then add it later. Maybe fix this in another PR?